src/components/pages/SingleArticle.jsx

```diff
@@ -0,0 +1,31 @@
+import Header from "../Header.jsx";
+import Nav from "../Nav.jsx";
+import {useParams} from "react-router-dom";
+import {useEffect, useState} from "react";
+
+
+function SingleArticle(){
+    const [article, setArticle] = useState({})
```

Comment:
Please include a loading state here and display a message while the article is loading.